### PR TITLE
Move 'Best practices' to a more prominent place

### PR DESCRIPTION
**Pull request by wagnerand**
Adding @atsay who requested this.

### CODE REVIEW

**Comment by atsay** [comment]
Cool, thanks!

File: guide/bestpractices.rst [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+==============
+
+Git and Github
+--------------
+
+- Create a separate pull request for each bug.
+- If you have multiple commits in one pull request, squash them into one. For complex patches, having multiple commits might make the review easier.
+- If you are working with multiple bugs at a time, make sure you create a separate branches for each bug.
+- If you need to make changes to the commit, perhaps after feedback, amend the original commit and comment in the pull request what you've changed.  Don't create a new pull request or commit.
+- Include  the bug number in your commit & pull request title. If possible  follow this style -`Some text description (bug XXXXXXX)`.
```
Comment (tofumatt) [inline guide/bestpractices.rst]: Nitpick, but two spaces after "Include" should just be one space.

Comment (tofumatt) [inline guide/bestpractices.rst]: After "possible" as well.

File: guide/bestpractices.rst [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Best practises
```
Comment (tofumatt) [inline guide/bestpractices.rst]: Spelled "Practises" here but "practices" in the URL.

File: reference/git_github.rst [hunk 3]
```diff
@@ -3,17 +3,6 @@ Git and Github
 
 This document describes some best practices for using Git and Github.
 
-Best practises
```
Comment (tofumatt) [inline reference/git_github.rst]: Could this be kept here using an include? It's still relevant here and it's not so bad having the same info twice if it's an include.

Comment (wagnerand) [inline reference/git_github.rst]: It could, but I expect that `bestpractises.rst` will cover more than just git & Github topic. I'll add a link.

**Comment by wagnerand** [comment]
I hope that link works, didn't know how to test it.

**Comment by tofumatt** [comment]
@wagnerand I think you can build the docs locally... but r+ it seems fine to me.

**Comment by wagnerand** [comment]
Not quite, but I figured it out.

**Comment by wagnerand** [comment]
Thank you!

**Comment by wagnerand** [comment]
This is ready to merge, could someone with permissions do that please?

**Comment by Pes2009k** [comment]
thank you

**Pull request merged**